README.md

```diff
@@ -64,3 +64,17 @@ Here are some web pages that you may need to refer to:
       - Click the download button underneath the header image to download a zip folder containing the datasets. I was going to focus on         the 2019.csv dataset. 
       - The categorical fields are Overall Rank, Country or Region, Score, GDP per Capita, Social Support, Healthy Life Expectancy,             Freedom to make life choices, Generosity, and Perceptions of Corruption. Each category from GDP per Capita to Perceptions of             Corruption has a number that contributes to each country’s happiness score which ranges from 0 – 10.
       - There are no numerical fields in this dataset.
+      
+   - **Abigail Hickey**
+    -*Tesla stock data from 2010 to 2020
```

Comment:
* Put a space after the dash to use the markdown bullet points. 
* You can get indented levels by just indenting the sub-level's dash. I recommend two-space indentations, but anything seems to work. So you won't need the single asterisks to show a topic heading.
* Take a look at my example and Brendans's entry for some ideas of things to include. I particularly would like for you to include whether it has both numeric and categorical columns, whether it relates to your major or minor, and which dataset you would prefer to work on.